.travis.yml

```diff
@@ -13,20 +9,48 @@ cache:
 env:
   global:
     - JACKRABBIT_VERSION=2.12.0
+    - SYMFONY__PHPCR__TRANSPORT=doctrinedbal
+    - TEST_FLAGS=""
+    - SYMFONY__DATABASE__DRIVER=pdo_mysql
+
+matrix:
+  include:
+    - php: 5.5
+    - php: 7.0
+      env: 
+        - SYMFONY__PHPCR__TRANSPORT=jackrabbit
+        # restart jackrabbit after each suite see: https://github.com/sulu-io/sulu/issues/2137
+        - TEST_FLAGS="--jackrabbit-restart"
+
+# Sqlite support: https://github.com/sulu/sulu/issues/2048
+# Postgres support: https://github.com/sulu/sulu/issues/2241
+#
+#    - php: 7.0
+#      env:
+#        - SYMFONY__DATABASE__DRIVER=pdo_sqlite
+#        - SYMFONY__DATABASE__PATH=test.sqlite
+#    - php: 7.0
+#      env: 
+#        - SYMFONY__DATABASE__DRIVER=pdo_pgsql
+#        - SYMFONY__DATABASE__USER=postgres
+#        - SYMFONY__DATABASE__PASSWORD=postgres
+#  fast_finish: true # do not wait for allowed failures, as they do not affect the build.
```

Comment:
Commented rather than deleted, as I think some of this is non-evident and it is useful for testing in the future.